invenio_app_ils/ext.py

```diff
@@ -298,6 +299,8 @@ def _register_signals(self, app):
         """Register signals."""
         register_circulation_signals()
         register_files_signals()
+        register_item_signals()
```

Comment:
Suggestion: we could maybe refactor this and have:
```
register_record_delete_signal()
```
and instead of `register_location_signals` have `register_record_update_signal()`.

Then, register the signal only once, in one method, and inside the method check what record. E.g.:

```
def register_record_delete_signal():
    after_record_delete.connect(record_delete_listener, weak=False)

def record_delete_listener(sender, *args, **kwargs):
    schema = record["$schema"]
    is_eitem = schema.endswith( current_app_ils.eitem_record_cls._schema)
    ... is_item ...
    if is_eitem....
    elif is_item ....
```

and

```
def register_record_update_signal():
    after_record_update.connect(record_update_listener, weak=False)

def record_update_listener(sender, *args, **kwargs):
    record = kwargs["record"]
    if isinstance(record, current_app_ils.location_record_cls):
        if app.config["EXTEND_LOANS_LOCATION_UPDATED"]:
            notify_location_updated(record["pid"])
```

Reply:
I was considering it, and I decided to split in the end to separate modules, because I thought it would be following the convention of the code residing in the each module (e-items handling in eitems folder etc) but it is not a strong preference, I can handle it on the top application level.

Reply:
in the end I implemented the hybrid, signals are registered in the respective modules, but handled by one common record delete listener